### PR TITLE
src: add total row to findjsobjects command

### PR DESCRIPTION
**Pull request by mmarchini**
Add a row with total objects count and total size to findjsobjects.

Demo: 
[asciicast](https://asciinema.org/a/147759)

### CODE REVIEW

**Comment by joyeecheung** [comment]
BTW: the CI is half-broken right now, needs https://github.com/nodejs/llnode/pull/144 and https://github.com/nodejs/github-bot/pull/158 to fix that, so ignore that yellow "Some checks haven’t completed yet".

File: src/llscan.cc [hunk 1]
```diff
@@ -61,7 +61,7 @@ bool FindObjectsCmd::DoExecute(SBDebugger d, char** cmd,
   std::sort(sorted_by_count.begin(), sorted_by_count.end(),
             TypeRecord::CompareInstanceCounts);
 
-  uint64_t total_objects = 0;
+  uint64_t total_objects = 0, total_size = 0;
```
Comment (indutny) [inline src/llscan.cc]: Minor nit: do you mind splitting this into two separate declarations?

Comment (mmarchini) [reply src/llscan.cc]: No problem. PR updated with this change

Comment (indutny) [reply src/llscan.cc]: Thank you!

**Pull request merged**